sw/otbn/code-snippets/mul256.S

```diff
@@ -0,0 +1,89 @@
+/* Copyright lowRISC contributors. */
```

Comment:
Please add the two new code snippets to the README in the `sw/otbn/code-snippets` directory.